### ISSUE TITLE
Monitoring the Rift:Sundered Chasm Reading

### ISSUE THREAD

**Issue by Excessdday** [dalaranwow/dalaran-wow#5466]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**:Trying to do the quest, Monitoring the Rift:Sundered Chasm, i was unable to complete the quest, due to not being able to use the item no matter where i was on the rift. It was just myself hunter and my pet. Level 70. 

**Expected behaviour**: I should be able to complete this quest on the location of the rift, 43.64, 30.63 and the surrounding area. This is the 2nd rift you need to Read.

**Steps to reproduce the problem**:

1. Logged onto 2 different charcters with the same effect.
2. Swam up and down the entire rift not just at the specified location.
3. 

**Include proofs for this behaviour**
http://www.wowhead.com/quest=11582/monitoring-the-rift-sundered-chasm

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Chenocide** [comment]
Make sure that you swim all the way down to the bottom. Quest worked all the times I've tried it.

**Comment by Golganeth** [comment]
Works fine and the credit is given as long as you are on the correct spot. Tested in-game right now just in case.

Closing
